assets/js/frontend/utils.js

```diff
@@ -0,0 +1,20 @@
+/* jshint esversion: 6 */
+/**
+ * Check if we're on mobile resolution.
+ *
+ * @returns {boolean}
+ */
+export const isMobile = function () {
+	var windowWidth = window.innerWidth;
+	return windowWidth <= 960;
+};
+
+export const httpGetAsync = function (theUrl, callback) {
+	let xmlHttp = new XMLHttpRequest();
+	xmlHttp.onload = function() {
+		if (xmlHttp.readyState === 4 && xmlHttp.status === 200)
+			callback(xmlHttp.response);
+	};
```

Comment:
we don't an error callback, at least we could show and error in the console when the request is unsuccessful